.github/workflows/crate-maintenance.yml

```diff
@@ -0,0 +1,18 @@
+name: Crate maintenance
+
+on:
+  schedule:
+    - cron: "14 3 * * 5" # every friday at 03:14
+  workflow_dispatch:
+
+jobs:
+  rust-maintain:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      # you should use a *pinned commit*:
```

Comment:
why is this?

Reply:
Just a reminder about software supply chain issues and auditing your dependencies: https://github.com/Swatinem/rust-maintain?tab=readme-ov-file#permissions--security :-)

As this action is running with hightened permissions.